libs/ngxtension/call-apply/src/call-apply.ts

```diff
@@ -23,7 +23,7 @@ const NOTHIS = !('Proxy' in window)
 	standalone: true,
 })
 export class CallPipe implements PipeTransform {
-	transform(value: any, args?: Function): any {
+	transform<T = any, R = any>(value: T, args?: (param?: T) => R): R {
```

Comment:
```suggestion
	transform<T = any, R = any>(value: T, args?: (param: T) => R): R {
```

If you make `param?: T` then it doesn't work because TypeScript will complain that `T | undefined is not assignable to T`

Reply:
Sorry my bad, I've just discovered and fixed this problem in my [stackbliz repro]() and forget to port back it to the code before commit 😜

Reply:
updated commit and PR 😉